src/SpinWaveTheory/SpinWaveTheory.jl

```diff
@@ -17,7 +17,9 @@ struct SpinWaveTheory
     sys   :: System
     s̃_mat :: Array{ComplexF64, 4}  # dipole operators
```

Comment:
I suppose we may add a list of observables here (for constructing the relevant `Avec` in intensities). But that can come after merging. (Doesn't apply to dipole mode.)